spec/oncocore_vs.txt

```diff
@@ -94,25 +94,22 @@ SCT#250724005 "Tumor marker measurement (procedure)"
 SCT#386344002 "Laboratory data interpretation (procedure)"
 //MTH#C0005516	"Biological markers"
 
-ValueSet:		CancerStageTimingVS
+ValueSet:		TimeOfClassificationVS
 //TODO: the value set is missing codes for recurrence and anatomical staging.
-//TODO: need new terms for post-therapeutic clinical and pathological staging.
-Description:	"When staging was done, relative to treatment events (qualifier)."
+Description:	"When staging was performed, relative to treatment events."
 SCT#385349001   "Clinical stage (observable entity)"
-//#cli		    "Clinical stage: Staging performed prior to surgery, based on evidence such as physical examination, imaging, and/or biopsy."
-SCT#405978005   "Pathologic stage (observable entity)"    //TODO_discussion: using SCT instead of local codes, we lose the definition. Also strange to use an observable in a value set.
-//#path         "Pathologic stage: Staging performed based on examination of tissue samples removed during surgery, in addition to physical examination and imaging."
+SCT#405978005   "Pathologic stage (observable entity)"
 //#postther_cli   "Post-therapeutic (post-neoadjuvant) clinical staging based on clinical examination after the first treatment with systemic drugs or radiation, when no surgery has been performed."
 //#postther_path  "Post-therapeutic (post-neoadjuvant) pathologic staging, based  on tissue samples removed during surgery after the first treatment with systemic drugs or radiation."
 
 ValueSet:   CancerStagingSystemVS   // replaces local value set in BrCa spec
 Description: "The system used to identify cancer/tumor stage"
 Includes codes descending from SCT#254292007 "Tumor staging (tumor staging)" //TODO: consider staging systems in NCI MTH and determine if there are gaps.
 
-ValueSet:   TNMTumorStagingSystemVS    // TODO: Need to include version of staging system (this is not sufficient)
-Description:    "System used for TNM classification. Includes the following codes:
+ValueSet:   TNMCancerStagingSystemVS    // TODO: Need to include version of staging system (this is not sufficient)
+Description:    "System used for staging, when staging uses TNM categories. Includes the following codes/staging systems:
```

Comment:
Awkward phrasing. "Suggest: TNM system used for staging, including version where applicable. Not all versions of staging systems may be explicitly represented."

Reply:
Would remove "Includes the following codes/staging systems".

Reply:
I would like to include the list because otherwise it won't be in the IG. Wording change is insignificant.